public/client.js

```diff
@@ -121,8 +123,11 @@ socket.on('Login_Status', function (data) {
 
 });
 
-
 /* ---------- Game Start --------- */
+function joinGameOne() {
+	socket.emit("joinGameOne");
+}
+
 function initGame() {
 	// Emitted upon 'Start' button click
 	socket.emit("initGame", true);
```

Comment:
we can put an if statement in this function to handle not enough players, and it'll throw an alert message on the client

``` function initGame() {
            let threshold_players = 2; // 2 players to start a game
            let num_players = document.getElementById("num_players_g1").innerHTML; 
            if (num_players) >= threshold_players) {
                    socket.emit("initGame", true)
             } else {
                     alert("Not enough players to start game");
              }